crates/bench/benches/uv.rs

```diff
@@ -157,9 +160,20 @@ mod resolver {
             PreviewMode::Disabled,
         );
 
+        let options = OptionsBuilder::new()
+            .exclude_newer(Some(
+                NaiveDate::from_ymd_opt(2024, 6, 20)
```

Comment:
Not sure if it'd be preferred here, but this is what we do in the tests https://github.com/astral-sh/uv/blob/e783a79955a3a4eb6a4c546f51f89e88b64047bb/crates/uv/tests/common/mod.rs#L26

Reply:
This is parsed by the CLI but you might want to use a a constant like this? Maybe use the same date?

Reply:
I wanted to avoid the string parsing because it would end up part of the benchmark.. probably negligible but I think it's fine.